[2024-12-08T13:10:36.346+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.insert_book_data manual__2024-12-08T13:10:24.570986+00:00 [queued]>
[2024-12-08T13:10:36.374+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.insert_book_data manual__2024-12-08T13:10:24.570986+00:00 [queued]>
[2024-12-08T13:10:36.375+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-08T13:10:36.396+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): insert_book_data> on 2024-12-08 13:10:24.570986+00:00
[2024-12-08T13:10:36.405+0000] {standard_task_runner.py:60} INFO - Started process 1043 to run task
[2024-12-08T13:10:36.411+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'insert_book_data', 'manual__2024-12-08T13:10:24.570986+00:00', '--job-id', '459', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpk6p796t9']
[2024-12-08T13:10:36.412+0000] {standard_task_runner.py:88} INFO - Job 459: Subtask insert_book_data
[2024-12-08T13:10:36.429+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-08T13:10:36.469+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.insert_book_data manual__2024-12-08T13:10:24.570986+00:00 [running]> on host 08c9c3bec828
[2024-12-08T13:10:36.604+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='insert_book_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-08T13:10:24.570986+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-08T13:10:24.570986+00:00'
[2024-12-08T13:10:36.714+0000] {logging_mixin.py:188} INFO -                           city  ...         sunset_time
0                     Toulouse  ... 2024-12-08 17:17:37
1     Arrondissement de Nantes  ... 2024-12-08 17:16:48
2   Arrondissement de Grenoble  ... 2024-12-08 16:54:55
3  Arrondissement de Marseille  ... 2024-12-08 17:02:18
4       Arrondissement de Lyon  ... 2024-12-08 16:57:30
5                     Montreal  ... 2024-12-08 16:11:03
6                     New York  ... 2024-12-08 16:28:38
7                      Conakry  ... 2024-12-08 18:34:24
8                        Dakar  ... 2024-12-08 18:40:28

[9 rows x 12 columns]
[2024-12-08T13:10:36.738+0000] {base.py:83} INFO - Using connection ID 'weather_database_connection' for task execution.
[2024-12-08T13:10:36.760+0000] {logging_mixin.py:188} INFO - 0    (Toulouse, light rain, 6.990000000000009, 2.84...
1    (Arrondissement de Nantes, overcast clouds, 7....
2    (Arrondissement de Grenoble, snow, -4.12999999...
3    (Arrondissement de Marseille, few clouds, 8.64...
4    (Arrondissement de Lyon, light rain, 4.5600000...
5    (Montreal, snow, -0.5799999999999841, -6.51999...
6    (New York, few clouds, 5.140000000000043, 0.81...
7    (Conakry, few clouds, 28.03000000000003, 27.76...
8    (Dakar, broken clouds, 26.450000000000045, 26....
dtype: object
[2024-12-08T13:10:36.765+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-08T13:10:36.788+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=insert_book_data, execution_date=20241208T131024, start_date=20241208T131036, end_date=20241208T131036
[2024-12-08T13:10:36.823+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-08T13:10:36.838+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
